shared/js/dialer/keypad.js

```diff
@@ -24,6 +25,7 @@ var KeypadManager = {
   _keypadSoundIsEnabled: false,
   _shortTone: false,
   _vibrationEnabled: false,
+  _simContactsList: [],
```

Comment:
I don't think we should cache these. It's used sufficiently infrequently that I think it's ok to reload over and over. The other issue here is that, as Tamara pointed out, if the contact list on the SIM changes, we won't reflect those changes here.